From: Mauricio Faria de Oliveira <mauricfo@linux.vnet.ibm.com>
Date: Thu, 25 Jan 2018 12:02:55 +0100
Subject: enable find multipaths

Forwarded: no
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1463046
--- a/libmultipath/defaults.h
+++ b/libmultipath/defaults.h
@@ -24,7 +24,7 @@
 #define DEFAULT_NO_PATH_RETRY	NO_PATH_RETRY_UNDEF
 #define DEFAULT_VERBOSITY	2
 #define DEFAULT_REASSIGN_MAPS	0
-#define DEFAULT_FIND_MULTIPATHS	FIND_MULTIPATHS_STRICT
+#define DEFAULT_FIND_MULTIPATHS	FIND_MULTIPATHS_ON
 #define DEFAULT_FAST_IO_FAIL	5
 #define DEFAULT_DEV_LOSS_TMO	600
 #define DEFAULT_RETAIN_HWHANDLER RETAIN_HWHANDLER_ON
